object-cache.php

```diff
@@ -769,18 +782,54 @@ public function __call( $name, $arguments ) {
 				case 'exists':
 					return false;
 			}
+
```

Comment:
Would still be super-helpful to be able to (optionally) log failures here. i.e. https://github.com/pantheon-systems/wp-redis/pull/5/files#diff-5a2b681f0d32956de83215ea32388177R755